releasenotes/notes/sbplx_optimizer-a27361fc4ac8de8c.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Support for SBPLX optimizer from NLopt library has been added.
+    SBPLX is a local gradient-free optimizer based on Nelder-Mead and
+    is expected to show better convergence behaviour.
+    Further information about this and all other optimizers can be found
+    :ref:`here<https://qiskit-community.github.io/qiskit-algorithms/apidocs/qiskit_algorithms.optimizers.html>`.
```

Comment:
```suggestion
    `in the API ref. here <https://qiskit-community.github.io/qiskit-algorithms/apidocs/qiskit_algorithms.optimizers.html>__`.
```
Its failing here as one can infer from reno in the message when its processing this. The link formatting needs a space between the text that will show and the link itself and :ref: is not needed for link like this and it needs the trailing __. I changed the text a little to be a bit more explicit too in the suggestion

> /home/runner/work/qiskit-algorithms/qiskit-algorithms/<reno.sphinxext current branch>:32: WARNING: undefined label: 'https://qiskit-community.github.io/qiskit-algorithms/apidocs/qiskit_algorithms.optimizers.html'